docs/LINUX.md

```diff
@@ -0,0 +1,121 @@
+# LogDNA Agent on Linux distributions
+
+The agent supports 64-bit (`x86_64`) Linux distributions and provides rpm and deb packages.
+
+## Installing
+
+### On Debian-based distributions
+
+```shell script
+echo "deb https://assets.logdna.com stable main" | sudo tee /etc/apt/sources.list.d/logdna.list
+wget -qO - https://assets.logdna.com/logdna.gpg | sudo apt-key add -
+sudo apt-get update
+sudo apt-get install logdna-agent
+```
+
+### On RPM-based distributions
+
+```shell script
+sudo rpm --import https://assets.logdna.com/logdna.gpg
+echo "[logdna]
+name=LogDNA packages
+baseurl=https://assets.logdna.com/el6/
+enabled=1
+gpgcheck=1
+gpgkey=https://assets.logdna.com/logdna.gpg" | sudo tee /etc/yum.repos.d/logdna.repo
+sudo yum -y install logdna-agent
+```
+
+# Usage
+
+You can start sending logs by setting the [ingestion key][ingestion-key]:
+
+```shell script
+logdna-agent -k <YOUR INGESTION KEY>
+```
+
+The agent expose short argument abbreviations for commonly used options (`-k`, `-t`, `-d` and `-c`).
+
+You can use `--help` flag to list all the command and environment variable options. For example with agent
+version `3.3.0`:
+
+```shell script
+$ logdna-agent --help
+LogDNA Agent 3.3.0
+A resource-efficient log collection agent that forwards logs to LogDNA.
+
+USAGE:
+    logdna-agent [OPTIONS]
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+    -c, --config <config>
+            The config filename [env: LOGDNA_CONFIG_FILE=]  [default: /etc/logdna/config.yaml]
+
+        --db-path <db-path>
+            The directory the agent will store it's state database. Note that the agent must have write access to the
+            directory and be a persistent volume. Defaults to "/var/lib/logdna-agent/" [env: LOGDNA_DB_PATH=]
+        --endpoint-path <endpoint-path>
+            The endpoint to forward logs to. Defaults to "/logs/agent" [env: LOGDNA_ENDPOINT=]
+        --exclude-regex <exclusion-regex>...
+            List of regex patterns to exclude files from monitoring [env: LOGDNA_EXCLUSION_REGEX_RULES=]
+        --exclude <exclusion-rules>...
+            List of glob patterns to exclude files from monitoring, to add to the default [env: LOGDNA_EXCLUSION_RULES=]
```

Comment:
"... to add to the default ----what exactly---? Does it mean "... to add to the default set of exclusion rules? Or to the default value... we need to qualify the term "default". ;-)